lib/slop/parser.rb

```diff
@@ -80,6 +80,13 @@ def parse(strings)
 
       @arguments += ignored_args
 
+      unused_options.each do |o|
+        if o.config[:required] && !suppress_errors?
```

Comment:
Perhaps the whole loop can be obviated when `!suppress_errors?`

Reply:
Very true. I'll fix that.